Extensions/old_sidebar.js

```diff
@@ -0,0 +1,71 @@
+//* TITLE Old Sidebar **//
+//* VERSION 1.1.0 **//
+//* DESCRIPTION Get the sidebar back **//
+//* DEVELOPER estufar **//
+//* FRAME false **//
+//* BETA false **//
+
+XKit.extensions.old_sidebar = new Object({
+	
+	running: false,
+	
+	preferences: {
+		"dashonly": {
+			text: "Only run on the dashboard",
+			default: false,
+			value: false
+		}
+	},
+	
+	run: function() {
+		XKit.tools.init_css("old_sidebar");
+		this.running = true;
+		
+		var movesidebar = true;
```

Comment:
Instead of having a variable movesidebar just do `` return; `` where you otherwise use `` movesidebar = false; ``